deployments/cognito-rds-s3/terraform/main.tf

```diff
@@ -103,6 +148,11 @@ module "eks_blueprints_kubernetes_addons" {
   enable_aws_efs_csi_driver = true
   enable_aws_fsx_csi_driver = true
 
+  nvidia_device_plugin_helm_config = {
```

Comment:
after looking at https://github.com/NVIDIA/k8s-device-plugin/tree/v0.12.3#deployment-via-helm, I take my [previous comment back](https://github.com/awslabs/kubeflow-manifests/pull/396#discussion_r975891622) about installing it in `kube-system`. Not sure why it is not uniform but since helm is recommended method to deploy, we can use as-is from blueprints

Reply:
done